examples/dotnet/Examples/FlexibleSyncExamples.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using Realms;
+using Realms.Sync;
+using MongoDB.Bson;
+using System.Linq;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+namespace Examples
+{
+    public class FlexibleSyncExamples
+    {
+        [Test]
+        public async Task TestUseFlexibleSync()
+        {
+            var app = App.Create("dotnet-flexible-wtzwc");
+            var user = await app.LogInAsync(Credentials.Anonymous());
+
+            // :code-block-start: open-a-flexible-synced-realm
+            var config = new FlexibleSyncConfiguration(app.CurrentUser);
+            var realm = Realm.GetInstance(config);
+            // :code-block-end:
+    
+
+            // :code-block-start: get-subscriptions
+            var subscriptions = realm.Subscriptions;
+            // :code-block-end:
+
+            // :code-block-start: update-subscriptions
+            subscriptions.Update(() =>
+            {
+                // subscribe to all long running tasks, and give the subscription the name 'longRunningTasksSubscription'
+                var longRunningTasksQuery = realm.All<MyTask>().Where(t => t.Status == "completed" && t.ProgressMinutes > 120 ); 
+                var longRunningTasksSubscriptionOptions = new SubscriptionOptions() { Name = "longRunningTasksSubscription" };
+                var longRunningTasksSubscription = subscriptions.Add(longRunningTasksQuery, longRunningTasksSubscriptionOptions);
```

Comment:
This can be simplified with the C# 10 syntax to:

```suggestion
               subscriptions.Add(longRunningTasksQuery, new() { Name = "longRunningTasks" });
```

There's no need to include `subscription` in the name, as it's fairly obvious that everything in the subscriptions collection is a subscription. There's also no need to assign the return value of `.Add()` if you're not going to use it.